docs/source/programming/GIN-repositories.md

```diff
@@ -0,0 +1,338 @@
+
+We recommend using [pooch](https://www.fatiando.org/pooch/latest/index.html) to easily download data from the GIN repo's URL. Pooch also has some other nice functionalities like caching the downloaded data, verifying cryptographic hashes or unzipping files upon download.
+
+## Other useful tips
+
+- To [unannex a file](https://gin.g-node.org/G-Node/Info/wiki/FAQ+Troubleshooting#how-to-unannex-files), aka remove a file from the GIN tracking before uploading:
+
+  ```
+  gin git annex unannex [path/filename]
+  ```
+
+- To stop tracking the GIN repo locally delete the `.git` directory
+
+:::{note}
+If in the GIN repo the files are locked, remember to unlock them before removing the `.git` directory! Otherwise we won't be able to delete the `.git/annex` content.
+:::
+
+- To delete a GIN repository but keep the git repo:
+
+  - delete the repository in the GIN server via the browser
+  - delete the GIN local repository with `git annex uninit`
+    - this command removes relevant bits in `.git/annex` and `.git/objects`, but some pre-commits may need to be edited by hand (see this [SO post](https://stackoverflow.com/questions/24447047/remove-git-annex-repository-from-file-tree)).
+
+## File locking
+
+[File locking](https://gin.g-node.org/G-Node/Info/wiki/GIN+CLI+Usage+Tutorial#file-locking) is an important point in GIN repos and git-annex which surprisingly comes up quite late in the GIN docs. Below, are the main ideas behind this.
+
+- Files in a GIN repo can be _locked_ or _unlocked_.
+
+- The lock state relates to the nature of the placeholder files we get in the working directory when we do `gin get <remote-repository-location>`:
+
+  - **on Unix-like systems**:
+    - if a file is _locked_, its corresponding placeholder file will be a _symlink_. These symlinks point to the annexed content (under `.git/annex/objects`). With the symlinks we can open the file but not modify it.
+    - If a file is _unlocked_, the placeholder file in the working directory is an _ASCII text file with a path_. The path is approximately where the content of the file will be downloaded to when we request it.
+  - **on Windows**:
+    - if a file is _locked_, the placeholder file is a _plain text file_ pointing to the content in the git annex.
+    - If a file is _unlocked_, the behaviour should be the same as in Unix-like systems (I haven't tested this directly).
+
+- The lock state of a file is _persistent_. This means that if I clone a GIN repo whose files are unlocked, I lock them in my local copy, and then upload that to the GIN server, the next time someone clones from the GIN repo the files they fetch will be locked.
+
+- Unlocked files can be edited. If the data is unlocked and the full content of the dataset is downloaded locally, the file in the working directory has content, and so does its copy under git annex.
+
+:::{caution}
+This doubles disk usage of files checked into the repo, but in exchange users can modify and revert files to previous commits.
```

Comment:
It's not 100% clear what this admonition refers to.

Reply:
what do you mean?